content/app/github/commit-page-helper.js

```diff
@@ -26,11 +26,18 @@ define("content/app/github/commit-page-helper", function(require, exports, modul
 				var files = CommitPageHelper.getAllFiles();
 
 				var file = null;
+				var maxArea = 0;
```

Comment:
Yep, we need to discuss this.
I'll make current file the one which intersects or is the closest one to the middle horizontal line through viewport. At least for now, and later we can find better approach.